code/modules/mod/mod_theme.dm

```diff
@@ -1360,11 +1339,8 @@
 
 /datum/mod_theme/administrative
 	name = "administrative"
-	desc = "A suit made of adminium. Who comes up with these stupid mineral names?"
-	extended_desc = "Yeah, okay, I guess you can call that an event. What I consider an event is something actually \
-		fun and engaging for the players- instead, most were sitting out, dead or gibbed, while the lucky few got to \
-		have all the fun. If this continues to be a pattern for your \"events\" (Admin Abuse) \
-		there will be an admin complaint. You have been warned."
+	desc = "A completely game-breaking suit made of compressed admin abuse. Naturally, it's got bullshit stats, doesn't drain any power at all and... for fuck's sake, does it really need a thousand module slots?!"
+	extended_desc = "someone go get the intern to write something here"
```

Comment:
```suggestion
	extended_desc = "Someone go get the intern to write something here."
```

Reply:
The grammar errors are intentional here.